src/components/NcListItem/NcListItem.vue

```diff
@@ -243,6 +243,17 @@
 						</div>
 					</div>
 
+					<div v-if="$slots.indicator && showDetails" class="list-item-content__secondary">
```

Comment:
What happens if there is an indicator slot, but `showDetails` is false? Then neither the slot nor `details` or `title` will be shown. This seems not correct.

Reply:
right